packages/ckeditor5-utils/src/index.ts

```diff
@@ -87,6 +87,7 @@ export { default as insertToPriorityArray } from './inserttopriorityarray';
 export { default as spliceArray } from './splicearray';
 
 export { default as uid } from './uid';
+export { default as verify } from './madewith';
```

Comment:
Please use a better name. What does it verify?

Reply:
Fixed.